catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/resource/download/ReliableResourceDownloader.java

```diff
@@ -177,7 +181,7 @@ public ResourceResponse setupDownload(Metacard metacard,
                 return resourceResponse;
             }
 
-            if (!resourceCache.isPending(key)) {
```

Comment:
That dependency has already been removed by @cmthom10 and @garrettfreibott. Probably need to sync up with them before we move forward with this PR.